### PR TITLE
Updated docs for dev/core#1077

### PR DESCRIPTION
**Pull request by sushantpaste**

### CODE REVIEW

**Comment by MikeyMJCO** [comment]
Please provide a description/some context around this docs change and why it's being made.

**Comment by sushantpaste** [comment]
It's related to https://lab.civicrm.org/dev/core/issues/1077 added example for using hook to alter reports links

File: docs/hooks/hook_civicrm_links.md [hunk 1]
```diff
@@ -112,6 +112,10 @@ function MODULENAME_civicrm_links($op, $objectName, $objectId, &$links, &$mask,
              // old extensions using 'title' will still work
           );
           break;
+        case 'view.report.links':
+          // disable copy,delete links.
```
Comment (MikeyMJCO) [inline docs/hooks/hook_civicrm_links.md]:
```suggestion
          // disable copy & delete links.
```

Comment (sushantpaste) [reply docs/hooks/hook_civicrm_links.md]: Added changes.

**Comment by MikeyMJCO** [comment]
Thanks very much. Merging!

**Pull request merged**